openquake/commonlib/calculators/base.py

```diff
@@ -332,3 +334,57 @@ def save_pik(self, result, **kw):
             with open(cache, 'w') as f:
                 cPickle.dump(self.risk_out, f)
         return self.risk_out
+
+
+# functions useful for the calculators ScenarioDamage and ScenarioRisk
+
+
+def get_gmfs(self):
```

Comment:
Here and in the following functions, please rename self -> calc (or other) and describe the param in the docstring